### PR TITLE
Gz new crypto

### PR DESCRIPTION
**Pull request by gzucca**
- Added type for the array containing all the cryptos in the global state. 
- Filtered userCryptos from appearing on search again.

Pending:
4) Update Cryptocurrency Data: creating an updating cycle so the value of user's cryptos are always updated.

### CODE REVIEW

File: src/components/CryptCard/index.tsx [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
+import {View} from 'react-native';
+import {useTheme} from 'styled-components';
 
-const CryptCard = (props: Props) => {
-  const theme = useTheme()
+const CryptCard = (props: CryptCardProps) => {
+  const theme = useTheme();
   const percentageChange = (percentNumber: number) => {
+    if (percentNumber === null) {
+      return <View></View>;
```
Comment (ThomasDevlabs) [inline src/components/CryptCard/index.tsx:23]: use <View />

File: src/components/CryptCards/index.tsx [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+const CryptCards = (props: CryptCardsProps) => (
   <View>
-    <CryptCard />
+    <ListScrollView>
+      {props.cryptosPassed.length > 0 &&
+        props.cryptosPassed.map(crypto => {
+          return (
+            <TouchableHighlight
+              key={crypto.id}
+              onPress={() =>
```
Comment (ThomasDevlabs) [inline src/components/CryptCards/index.tsx:15]: move to a function outside

File: src/components/CryptCards/index.tsx [hunk 3]
```diff
@@ -1,11 +1,36 @@
-import {View} from 'react-native';
+import {TouchableHighlight, View} from 'react-native';
 import React from 'react';
 import CryptCard from '../CryptCard';
+import {ListScrollView} from './styles';
+import {CryptCardsProps} from '../../../types';
 
-const CryptCards = () => (
+const CryptCards = (props: CryptCardsProps) => (
```
Comment (ThomasDevlabs) [inline src/components/CryptCards/index.tsx:7]: use props destructuring

File: src/pages/Search/index.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+  const showAlert = () =>
+    Alert.alert(
+      'No cryptocurrency found',
+      'Please enter the full name or ticker symbol. E.g.: "Bitcoin" or "BTC"',
+    );
+
+  const handleSaveCrypto = () => {
+    const check = globalState.searchResult.length === 1;
+    if (check === true) {
```
Comment (ThomasDevlabs) [inline src/pages/Search/index.tsx:39]: use if (check)

File: src/components/CryptCards/index.tsx [hunk 5]
```diff
@@ -1,36 +1,58 @@
...
+const CryptCards = ({
+  cryptosPassed,
+  setSelected,
+  selected,
+}: CryptCardsProps) => {
+  const handleSelected = (id: string) => {
+    selected === id ? setSelected('') : setSelected(id);
+  };
+
+  const Item = ({
```
Comment (tomasdematos) [inline src/components/CryptCards/index.tsx:16]: move to another file if you declare a component for individual cards.

File: src/components/CryptCards/index.tsx [hunk 6]
```diff
@@ -1,36 +1,58 @@
...
+      percentChange24hs={item.percentChange24hs}
+      onSelect={item.onSelect}
+    />
+  );
+
+  return (
+    <ListFlatList
+      data={cryptosPassed}
+      renderItem={renderItem}
+      keyExtractor={(item: any) => item.id}
```
Comment (tomasdematos) [inline src/components/CryptCards/index.tsx:54]: don't use any

**Pull request merged**